### ISSUE TITLE
The Symfony Installer is not compatible with Symfony 4.x or newer versions.

### ISSUE THREAD

**Issue by Olafxso** [Haehnchen/idea-php-symfony2-plugin#1334]
Creating a new project in PhpStorm with Symfony Version 4.3.2 (latest) failed because: 
he Symfony Installer is not compatible with Symfony 4.x or newer versions.  
			Run this other command to install Symfony using Composer instead:            
			composer create-project symfony/skeleton .checkout                           
			new <directory> [<version>]

Am I doing something wrong or is this a known issue?

**Comment by Haehnchen** [comment]
yes. but there is already a pr https://github.com/Haehnchen/idea-php-symfony2-plugin/pull/1330, but not finally done ;)

**Comment by Michigras** [comment]
Any update on this getting fixed?

**Comment by WarBorg** [comment]
I would also like to know if this has been fixed yet ?
